thriller-bindings/examples/gemm/gemm_g2r.py

```diff
@@ -45,37 +47,35 @@
     print(gB)
     print(gC)
 
-    MemoryLevel = pythriller.PyMemoryLevel.Register
-    RegGraph = pythriller.PyGraph()
+    RegGraph = Graph()
 
-    NodeA = pythriller.PyNode(rA)
-    NodeB = pythriller.PyNode(rB)
-    NodeAcc = pythriller.PyNode(acc)
+    NodeA = Node.tensor(rA)
+    NodeB = Node.tensor(rB)
+    NodeAcc = Node.tensor(acc)
 
-    GemmNode = pythriller.PyNode.gemm(NodeA, NodeB, NodeAcc)
+    GemmNode = Node.gemm(NodeA, NodeB, NodeAcc)
 
-    LoopIter = pythriller.IterationVar('i', (0, 4))
+    LoopIter = IterationVar('i', (0, 4))
 
     access_dims = [1]
 
-    AccessMap = pythriller.AccessMap(
+    AccessMap = AccessMap(
         access_dims, [[[1]], [[0]]], [[0], [10]], [LoopIter])
 
-    EdgeA_Gemm = pythriller.PyEdge(NodeA, GemmNode)
-    EdgeB_GEMM = pythriller.PyEdge(NodeB, GemmNode)
-    EdgeGemm_Acc = pythriller.PyEdge(GemmNode, NodeAcc)
+    EdgeA_Gemm = Edge(NodeA, GemmNode)
+    EdgeB_GEMM = Edge(NodeB, GemmNode)
```

Comment:
`EdgeA_gemm` and `EdgeB_GEMM` have different naming style, is this intentional?

Reply:
fixed.